pypeit/metadata.py

```diff
@@ -348,8 +348,6 @@ def convert_time(self, in_time, date=None):
         # Convert from an astropy.Time format
         if self.spectrograph.timeunit in time.Time.FORMATS.keys():
             if date is not None:
```

Comment:
I think this barfs for some instruments because `date` or `time` aren't strings.  I don't remember when this happens.  I wouldn't worry about it now, but we just need to keep an eye out.  I think I left these lines in there; thanks for removing them!

Reply:
np